activities/tool-management/nextcloud.md

```diff
@@ -0,0 +1,21 @@
+---
+type: resource
+---
+
+# Nextcloud
+
+Nextcloud is a suite of client-server software for creating and using file hosting services. Nextcloud is free and open-source. We are currenty using Nextcloud for file sharing with our community and the general public.
+
+## Using Nextcloud
+
+Staff are issued a Nextcloud account when they join. Nextcloud publishes [extensive support documentation](https://docs.nextcloud.com/).
+
+## Nextcloud management
+
+Nextcloud (user) administration is managed by the Operations Coordinator. The Operations Coordinator as well as the Chief Executive have full adminstrative privileges. The Operations Coordinator is responsible for user management, folder management, customization and making sure Nextcloud stays up-to-date. She also ensures proper on-boarding and off-boarding policies are in place and enforced.
```

Comment:
This seems correct, but perhaps the pronoun could be generalized to be more timeless?

Reply:
also a good point, I'll update it - thanks